Library/Homebrew/test/pathname_spec.rb

```diff
@@ -118,6 +118,7 @@
 
   describe "#ensure_writable" do
     it "makes a file writable and restores permissions afterwards" do
+      skip "User is root." if Process.euid == 0
```

Comment:
`User is root so everything is writable`.

Reply:
Fixed.